### DISCUSSION TITLE
how to start

### DISCUSSION THREAD

**Discussion by felpsgx** [ItsPi3141/discord-fake-avatar-decorations#8] [General]
Good afternoon, I wanted to know how to turn it on without it always turning off?

**Comment by ItsPi3141** [comment]
Can you please be more descriptive? What is turning off? What steps can be taken to reproduce this issue?

**Reply by felpsgx** [reply]
Então eu queria saber como que inicia o projeto depois de baixar ele pra área de trabalho queria saber como liga ele como liga o site.

**Reply by ItsPi3141** [reply]
You need to host it online with a web hosting service such as Vercel or Netlify. You can also just use the website here: https://discord-decorations.vercel.app/
